src/main/java/de/tum/in/www1/artemis/service/SubmissionService.java

```diff
@@ -314,14 +314,59 @@ public Result saveNewEmptyResult(Submission submission) {
         return result;
     }
 
+    /**
+     * Copy Feedbacks from one Result to a another Result
+     * @param newResult new result to copy feedback to
+     * @param oldResult old result to copy feedback from
+     * @return the list of newly created feedbacks
```

Comment:
This is more or less nitpicking (so feel free to ignore if you think this is unimportant: If `newResult` already has feedbacks the return value would include those too in addition to the created feedbacks. (Maybe we just remove the return value?)

Reply:
You're right that is not really necessary here, but as we generally try to return the value of things after saving them in the DB I think it's ok if we leave it like it is